app/src/main/java/com/standuptracker/dao/IPersonDAO.kt

```diff
@@ -6,7 +6,9 @@ import retrofit2.http.GET
 
 interface IPersonDAO {
 
-    // set json endpoint
+    /*
+     * Sets the JSON endpoint
```

Comment:
Changed this to a JavaDoc Comment

Reply:
For proper KDoc (as we call it in Kotlin), move it above the interface line.  JavaDoc/KDoc should be above a class or interface.